### DISCUSSION TITLE
有部分已经不能用了

### DISCUSSION THREAD

**Discussion by Phenix93** [docker-practice/docker-registry-cn-mirror-test#15] [General]
ustc的源已经重定向到了ali，但是阿里已经不维护了，好几个月没同步了。另外可以试试daocloud.io的服务，我目前用起来还行，跟ali一样要注册

**Comment by khs1994** [comment]
根据最新镜像的测试结果确实阿里云的不能使用了，尝试 @ 一下权威人士确认一下。
